src/components/Products/ProductsList.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+  <div
+    class="product-list"
+    v-bind:style="[
+      false
+        ? 'grid-template-columns: 1fr'
+        : 'grid-template-columns: 1fr 1fr 1fr'
+    ]"
+  >
+    <products-list-item
+      v-for="(item, index) in items"
```

Comment:
what if items are empty? searching for object attributes (like .name below) will cause errors
its easly fixed when you can add v-if="items" for whole component so it only renders when list is here